contrib/github-merge.sh

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
```

Comment:
another nitpick: could you make the script executable (such that chmod isn't needed after cloning) (signature_check.sh is ok)